src/MongoDB.EntityFrameworkCore/Extensions/MongoPropertyBuilderExtensions.cs

```diff
@@ -53,65 +54,127 @@ public static class MongoPropertyBuilderExtensions
         => (PropertyBuilder<TProperty>)HasElementName((PropertyBuilder)propertyBuilder, name);
 
     /// <summary>
-    /// Configures the <see cref="DateTimeKind"/> for the property.
+    /// Configures the document element that the property is mapped to when targeting MongoDB.
+    /// If an empty string is supplied then the property will not be persisted.
```

Comment:
Although not recommended, an empty string is a valid field name in MongoDB. `null` however is not a valid field name. Is an empty string not mapping a property an EF convention or something that we are introducing?

Reply:
This was just improving the documentation. EF uses this convention in other providers. If we want to support writing to an element with an empty name we may be able to that but we'd need to create a new ticket and figure out an alternative marker for unmapped properties.